webapp/pnpm.CMD

```diff
@@ -0,0 +1,2 @@
+@SETLOCAL
+@"C:\Program Files\nodejs\node.exe"  "%~dp0\..\..\..\..\AppData\Roaming\npm\pnpm-global\5\node_modules\pnpm\bin\pnpm.cjs" %*
```

Comment:
Hmm did you mean to commit these three pnpm files? And do you need them in this folder? Seems a bit odd to me they're in this folder, but I also rarely use Windows to write code in any meaningful way (I've got a Macbook right now and used to use Ubuntu) I'd ask Akshay or Signe if you're not sure and work it out together

If you still need them:

I'd Google how to remove them from `git` but not your file system and then add them to the `.gitignore` file with a comment explaining why.
If removing them from git is being weird you could copy them elsewhere, then `git rm $FILENAME` commit the delete then re-add them to this folder.

Reply:
I didn't install PNPM globally (thanks to Signe for helping figure this out). I reinstalled PNPM globally and removed the 3 PNPM files